### ISSUE TITLE
The estimate_keyword_traffic example doesn't work because No type 'Keyword' in namespace

### ISSUE THREAD

**Issue by rsmichaeldunn** [googleads/googleads-python-lib#270]
examples/adwords/v201802/optimization/estimate_keyword_traffic.py doesn't seem to work in python 3.6.  It follows the [documentation](https://developers.google.com/adwords/api/docs/guides/traffic-estimator-service) very closely, but when running the example, the result is 
```
zeep.exceptions.LookupError: No type 'Keyword' in namespace https://adwords.google.com/api/adwords/o/v201802.
Available types are: {https://adwords.google.com/api/adwords/o/v201802}AdGroupEstimate,
{https://adwords.google.com/api/adwords/o/v201802}AdGroupEstimateRequest,
{https://adwords.google.com/api/adwords/o/v201802}CampaignEstimate,
{https://adwords.google.com/api/adwords/o/v201802}CampaignEstimateRequest,
{https://adwords.google.com/api/adwords/o/v201802}CurrencyCodeError,
{https://adwords.google.com/api/adwords/o/v201802}Estimate,
{https://adwords.google.com/api/adwords/o/v201802}EstimateRequest,
{https://adwords.google.com/api/adwords/o/v201802}KeywordEstimate,
{https://adwords.google.com/api/adwords/o/v201802}KeywordEstimateRequest,
{https://adwords.google.com/api/adwords/o/v201802}PlatformCampaignEstimate,
{https://adwords.google.com/api/adwords/o/v201802}StatsEstimate,
{https://adwords.google.com/api/adwords/o/v201802}TrafficEstimatorError,
{https://adwords.google.com/api/adwords/o/v201802}TrafficEstimatorResult,
{https://adwords.google.com/api/adwords/o/v201802}TrafficEstimatorSelector,
{https://adwords.google.com/api/adwords/o/v201802}CurrencyCodeError.Reason,
{https://adwords.google.com/api/adwords/o/v201802}TrafficEstimatorError.Reason
```

**Comment by msaniscalchi** [comment]
Hello,

Thanks for the report! The underlying cause appears to be that Zeep is looking in the wrong namespace for the `Keyword` complexType, or perhaps just not looking in the correct one at all.

It would be helpful if you could provide the stracktrace that appears when running that example; we'd like to see exactly where this occurs.

Regards,
Mark

**Comment by grivescorbett** [comment]
Fixed in 11.0.1, please try again! :)

**Comment by rsmichaeldunn** [comment]
Looks good to me.  Thanks for the quick fix!
